Generals/Code/GameEngine/Include/Common/urllaunch.h

```diff
@@ -20,6 +20,8 @@
 #ifndef URLLAUNCH_H
 #define URLLAUNCH_H
 
+#include <windows.h>
```

Comment:
Is this meant to be here? I do not understand why these headers are necessary. When adding enum types, no new includes should be necessary.

Reply:
excluded